packages/mobile/locales/en-US/fiatExchangeFlow.json

```diff
@@ -0,0 +1,11 @@
+{
+  "add_funds": "Add funds",
+  "cash_out": "Cash out",
```

Comment:
any reason these are snake_case and the rest are Camel?

Reply:
My fault, will do camelCase! Good catch!